src/dirs.proj

```diff
@@ -26,17 +42,10 @@
     <!-- Explicity set the OutDir as it is used by the packaging targets -->
     <OutDir>$(BaseOutputPathWithConfig)</OutDir>
   </PropertyGroup>
-
+  
   <Import Project="$(ToolsDir)packages.targets" Condition="Exists('$(ToolsDir)packages.targets')" />
   <Target Name="BuidDirPackages" AfterTargets="Build" DependsOnTargets="BuildPackages" />
 
-  <ItemGroup>
-    <DisabledTestAssembly Include="System.Threading.Tasks.Dataflow.Tests" />
-    <DisabledTestAssembly Include="System.Linq.Parallel.Tests" />
-
-    <_SkipTestAssemblies Include="@(DisabledTestAssembly)" />
-  </ItemGroup>
```

Comment:
Sorry they're not, I did this to test them myself against the changes.  I'll update the PR to disable them and try and get another PR to reenable them (after talking to @mellinoe, PLINQ test owner)